tests/BrowsershotTest.php

```diff
@@ -910,4 +910,24 @@ public function it_can_add_a_wait_for_function_to_puppeteer()
             ],
         ], $command);
     }
+
+    /** @test **/
+    public function it_can_send_post_parameters_and_get_the_body_html()
+    {
+        $html = Browsershot::url('http://example.com')
+            ->usePost(['hello' => 'hi there!', 'is it me' => 'you are looking for?'])
+            ->bodyHtml();
+
+        $this->assertContains('<h1>Example Domain</h1>', $html);
```

Comment:
How does this output prove that `usePost` is actually working?

Reply:
There's currently no way of proving this using example.com as domain. We'd have to use something else like httpbin.org or another http post service that returns the post values. If that's possible I'll customize the 2 tests.